### ISSUE TITLE
Samples are not always played completely during validation

### ISSUE THREAD

**Issue by jf99** [mozilla/voice-web#1075]
Sometimes, you can already press play and then the voice recording plays for a few seconds, but then gets aborted. My initial guess would be that this happens, when the mp3 file has loaded only partially. Apparently, the common voice servers have difficulties handling the heavy load, again. ~Pressing stop and waiting a few seconds before pressing play again helps sometimes.~ Switching to another language and vaildate one or two sentences there usually helps.

I erroneously classified some recordings as "Nein" because I thought the recording was incomplete.

**Comment by Gregoor** [comment]
Thanks for noticing! Do you have a more-or-less-surefire way of reproducing it? Couldn't see it in my tests.

From what I can see the bug would either be in our Frontend, Browser or AWS (in descending order of likeliness). At least can't be our backend, as the clips are directly loaded from AWS.

**Comment by jf99** [comment]
Unfortunately not. Currently it doesn't happen anymore. But today from about 8-10 am, it was happening often. This was also when the sentences often took long to load.

**Comment by Gregoor** [comment]
I guess another problem source I neglected to mention is that users just clicked stop too early. Though given your description of a timeframe where it happened, that explanation seems unlikely. But that also Frontend and Browser an unlikely explanation. Leaves AWS?
Quite the puzzle we have on our hands 😕

**Comment by jf99** [comment]
In this case, it's definitely not the users who clicked stop too early. That's what I first thought, as well and then I voted for "Nein". But after some time, I noticed that this happened too often and that it happened with recordings that I already heard before which were correct.

And like I said: A workaround was to switch to another language, validate a few sentences there and then switch back. Et voilà, the same German recording suddenly played from beginning to the end.

Very weird, I agree. Let's see if it happens again the next days.
